indico/htdocs/js/indico/jquery/paper_email_settings_widget.js

```diff
@@ -0,0 +1,75 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2017 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function($) {
+    'use strict';
+
+    $.widget('indico.paperemailsettingswidget', {
+        options: {
+            containerElement: null,
+            multipleRecipientOptions: [
+                'notify_on_added_to_event', 'notify_on_assigned_contrib', 'notify_on_paper_submission'
+            ],
+            singleRecipientOptions: [
+                'notify_judge_on_review', 'notify_author_on_judgment'
+            ]
+        },
+
+        initCheckboxes: function(data) {
+            var self = this;
```

Comment:
then `var $container = this.options.containerElement;` should probably be changed to use self ;)